src/desktop/apps/auction/components/layout/ConfirmRegistrationModal.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { useEffect, useState } from "react"
+import { Modal } from "reaction/Components/Modal/Modal"
+import { Serif, Button, CheckCircleIcon, Theme, Box } from "@artsy/palette"
+import { connect } from "react-redux"
+
+const _ConfirmRegistrationModal = ({ me }) => {
+  useEffect(() => {
+    const replaceModalTriggerPath = location.pathname.replace(
+      "/confirm-registration",
+      ""
+    )
+    history.replaceState({}, document.title, replaceModalTriggerPath)
+  }, [])
+  if (!(me && me.bidders && me.bidders.length)) return null
```

Comment:
7-13: Remove the `/confirm-registration` portion of the path which triggered this modal.
14: If for some reason there is no user or they are not registered, render nothing.